### DISCUSSION TITLE
New style idea, "TextFlow"

### DISCUSSION THREAD

**Discussion by agoodfellow123** [dialogic-godot/dialogic#2149] [Ideas]
Hello, I was working on a game idea and made a specific UI, kind of similar to Inkle, unfortunately, my design was not compatible with Dialogic. It's a cool concept of design and I think Dialogic can benefit from having a style such as this one. If you are interested here is the code: https://github.com/agoodfellow123/gdstyle

It basically looks like this:
[sc.webm](https://github.com/dialogic-godot/dialogic/assets/36235265/b2f4617a-5a70-42db-9b3a-7266f6e0a74d)

**Comment by Jowan-Spooner** [comment]
Hey thanks. Something like this has been considered before, and is likely possible, but it has some draw-backs, because these url "buttons" ONLY work with the mouse and are otherwise quite user-unfriendly (for example for controlers, keyboard, etc). So while it's a fun feature for some games, it's a bit to limited to be a solution for most games I fear. 

That shouldn't stop us from implementing it though or at least playing around with it ;)

**Reply by agoodfellow123** [reply]
Hey, thank you for giving it a thought! If you need me, I will help you any way I can. I'm not that good at programming, but I would like to try if someone can show me the ropes around.
